raycicmd/build_id.go

```diff
@@ -0,0 +1,28 @@
+package raycicmd
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+)
+
+func makeBuildID(envs Envs) (string, error) {
```

Comment:
Is this referring to the buildkite build? Why user can also be a build id?

Reply:
removed.

I was thinking allowing user to run locally. but since it is just creating pipeline definitions now, I will just remove it.